sdk/tests/deqp/functional/gles3/es3fTransformFeedbackTests.js

```diff
@@ -939,8 +938,6 @@ goog.scope(function() {
 
         this.m_transformFeedback = gl.createTransformFeedback();
 
-        es3fTransformFeedbackTests.GLU_EXPECT_NO_ERROR(gl, gl.getError(), 'es3fTransformFeedbackTests.init');
```

Comment:
Error checking is now implemented in a context wrapper, WebGLTestUtils.create3DContextWithWrapperThatThrowsOnGLError. This makes error checking in the tests obsolete, so we can clean it up and make the code more readable.